spinalcordtoolbox/scripts/sct_compute_snr.py

```diff
@@ -120,6 +126,13 @@ def main(argv=None):
         index_vol_user = arguments.vol
     else:
         index_vol_user = ''
+    
+    #Added function for text file 
+    if arguments.o is not None:
+        file_name = arguments.o
+    else:
+        file_name = None
+
```

Comment:
Nitpick, but this is a simpler way of writing this section:

```suggestion
    file_name = arguments.o
```

Because the `None` code is redundant (`if arguments.o is None`, then the result will be the same).